apps/nextjs-website/src/_contents/products.ts

```diff
@@ -74,4 +74,10 @@ export const guides = [
   ...sendGuides,
 ];
 
+const gitBookContents = [...tutorials, ...guides];
```

Comment:
What about to put a comment here to explain why the `gitBookPagesWithTitle` is needed?